samples/apps/copilot-chat-app/scripts/README.md

```diff
@@ -1,45 +1,85 @@
-# Copilot Chat Setup Scripts
-
-> The PowerShell scripts in this directory require [PowerShell Core 6 or higher](https://github.com/PowerShell/PowerShell#get-powershell).
+# Copilot Chat Setup Scripts (local deployment)
 
 ## Before You Begin
-To run these scripts, you will need the following:
-- *REACT_APP_CLIENT_ID*
-  - This is the client ID (also known as application ID) associated with your Azure Active Directory (AAD) application registration, which you can find in the Azure portal.
-- *MY_AZUREOPENAI_OR_OPENAI_KEY*
+To run Copilot Chat, you will need the following:
+- *Application ID*
+  - This is the Client ID (i.e., Application ID) associated with your Azure Active Directory (AAD) application registration, which you can find in the Azure portal.
+- *Tenant ID*
+  - This is the Tenant ID (i.e., Directory ID) associated with your AAD app registration.
+  [Learn more about possible values for this parameter](https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration#authority).
```

Comment:
We don't actually ask for Tenant in the instructions below. Should we remove this?